pkgs/development/python-modules/cryptography/40.nix

```diff
@@ -0,0 +1,112 @@
+{ lib
+, stdenv
+, callPackage
+, buildPythonPackage
+, fetchPypi
+, rustPlatform
+, cargo
+, rustc
+, setuptoolsRustBuildHook
+, openssl
+, Security
+, isPyPy
+, cffi
+, pkg-config
+, pytestCheckHook
+, pytest-subtests
+, pythonOlder
+, pretend
+, libiconv
+, libxcrypt
+, iso8601
+, py
+, pytz
+, hypothesis
+}:
+
+let
+  cryptography-vectors = callPackage ./40_vectors.nix { };
+in
+buildPythonPackage rec {
+  pname = "cryptography";
+  version = "40.0.1"; # Also update the hash in vectors.nix
```

Comment:
Why not 40.0.2?

Reply:
Only `40.0.1` was in nixpkgs, so I picked the version that was known to work there. The only thing `40.0.2` has on its changelog is compatibility with a newer version of LibreSSL, which is not important to the backports for Ceph, and I'd prefer to change as little as possible.